doc/tutorials/understanding-error-messages.md

```diff
@@ -0,0 +1,48 @@
+# Understanding Error Messages
+
+If you want to use Elektra for any of your projects you will sooner or later run into errors. These can happen
```

Comment:
> If you want to use Elektra for any of your projects you will sooner or later run into errors.

I did not know we make this guarantee.

Reply:
It happens quite easily for me ;) Should I change it to "you may sooner or late" ?

Reply:
Yes, the sentence is a bit weird. Simply start with the second sentence that there are many different situations where errors might occur.

Reply:
Is the sentence at all relevant to understanding of error messages?

A better start for the tutorial might be.. "Errors can happen when trying to mount with insufficient permissions, [...]. The error message gives information on what the problem is." (and possibly how to resolve the problem?)

Regarding the initial sentence it might be better to open issues and give insight on how we can improve the situation. To me such a message does not seem to help the user.